twitter.css

```diff
@@ -0,0 +1,316 @@
+*{
+    box-sizing: border-box;
+}
+body{ margin: 0;
+}
+h2{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+/* LAY OUT OF PAGE*/
+.main-grid {
+    display: grid;
+    grid-template: ". centered ." auto / 1fr auto 1fr;
+}
+.two-row-grid{
+    display: grid;
+    grid-template-rows: 50px auto;
+}
+.grid-auto__column {
+    display: grid;
+    grid-auto-flow: column;
+  
+    gap: 2px;
+  }
+.center-wrapper{
+    max-width: 1250px;
+    margin: 0 auto
+}
+.three-column-grid{
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+}
+
+.three-row-grid{
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+}
+
+.two-column-grid__expand-two {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    column-gap: 18px;
+}
+.three-column-grid__expand-two {
+    display: grid;
+    grid-template-columns: 50px auto 50px;
+    column-gap: 5px;
+    padding: 10px;
+  }
+.three-row-grid__main{
+    display: grid;
+    grid-template-rows: 50px auto 1fr;
+
+}
+.two-column-grid{
+    display: grid;
+    grid-template-columns: 1fr auto;
+    column-gap: 18px;
+}
+
+.nav-headers > *{
+    min-width: auto;
+    min-height: auto;
+}
+.nav-headers{
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+font-size: 20px
+}
+.main-body > *{
+    padding: 2px;
+}
+/* Left Nav Bar*/
```

Comment:
I like how you separated the code into sections and its utility, it's easier to read the code